### DISCUSSION TITLE
Copier/coller les textes des chemins des propriétés et/ou additionnelles

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#595] [Ideas]
Bonjour,

Serai t'il possible de pouvoir copier les textes des chemins des propriétés ou additionnelles (exemple: properties.opacity , properties.visible , ...) ?

Pour éviter de faire des erreurs d'écriture dans les script d'évènements.

Merci d'avance ;)

**Comment by Raphwit** [comment]
Bonne idée, on ajoute à la liste, merci !

**Reply by Raphwit** [reply]
![image](https://github.com/witsa/synapps/assets/93767525/dba927dd-ef64-4425-974d-8ce15c86028d)

Je tease, pour la prochaine MAJ 😉
